### ISSUE TITLE
setup guide details !!

### ISSUE THREAD

**Issue by mukalaaa** [DirtyRacer1337/Jellyfin.Plugin.PhoenixAdult#3]
First thanks a million for doing this, it has been a dream coming true to have PhoenixAdult plugin support for Jellyfin / Emby

I tried to test your plugin with my huge library on my server but I had no idea how to get your plugin built !!
 
Clone or download this repository
**Ensure you have .NET Core SDK setup and installed**     "do you have a link where we can download this"

**Build plugin with following command**. (what exactly do you mean by that, I know CMD and terminal and commands but I dont understand what are the necessary steps for building the plugin)
dotnet publish --configuration Release --output bin

Place the resulting file in the plugins folder under the program data directory or inside the portable install directory

basically, what I am asking for, a more detailed step by step guide to set this one up please
one quick question too, does this work with emby or just jellyfin?

**Comment by DirtyRacer1337** [comment]
1. First link from the search, https://dotnet.microsoft.com/download
2. Build command, https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-build
Just run `dotnet publish --configuration Release --output bin` in project folder
3. Plugin install, https://jellyfin.org/docs/general/server/plugins/index.html

I tested it on Jellyfin, don't know is it compatible.
Also notice, it's only 2 sites available.

**Comment by mukalaaa** [comment]
Ok thanks for your reply 

I followed your instructions but I did not get a single file, I got 3 files 
Jellyfin.Plugin.PhoenixAdult.deps
Jellyfin.Plugin.PhoenixAdult.dll
Jellyfin.Plugin.PhoenixAdult.pdb

which one should I put in the plugins folder?

**Comment by mukalaaa** [comment]
Ok I think I figured it

I noticed that all the plugins that I already have are dll files so i moved the following file into jellyfin plugins folder
Jellyfin.Plugin.PhoenixAdult.dll ( I did not move anything else except for the mentioned dll file )

Now, I see the plugin in the jellyfin dashboard 

I added one brazzers file to test but it could not identify it 
the scene that I added is brazzers.20.05.26.rebecca.more.just.one.clit.away

I am almost there but not really sure whats wrong!! can you help me?

**Comment by DirtyRacer1337** [comment]
Look at `Required Libraries`, you need put them too

**Comment by mukalaaa** [comment]
where is that located? 
Because I dont see it in the downloaded code from github. I only see 
docs folder
Jellyfin.Plugin.PhoenixAdult folder 
and the rest are all files 

I dont see Required Libraries!!

**Comment by DirtyRacer1337** [comment]
Build with `dotnet publish --configuration Release --output bin` and you get all what you need

**Comment by mukalaaa** [comment]
Ok, thanks for your reply but I am sure something is missing and i am sure it is on my side so please allow me to share my steps with you 

I downloaded the  .NET Core SDK and installed it
I downloaded Jellyfin.Plugin.PhoenixAdult-master.zip from github
I unzipped the Jellyfin.Plugin.PhoenixAdult-master file and put it on my desktop 
I opened CMD
typed the following command: cd C:\Users\eilia\Desktop\Jellyfin.Plugin.PhoenixAdult-master 
I hit Enter and now i am in the directory of the project 
typed the following command: dotnet publish --configuration Release --output bin
CMD shows me many lines, some are yellow and others are white (please look at the attachment to see it)

I get:
docs folder
Jellyfin.Plugin.PhoenixAdult folder
and the rest are all files

no required libraries or anything else

Thanks for your patience and assistance
[CMD screen.zip](https://github.com/DirtyRacer1337/Jellyfin.Plugin.PhoenixAdult/files/4716344/CMD.screen.zip)

**Comment by DirtyRacer1337** [comment]
Everything should be in `bin` directory

**Comment by mukalaaa** [comment]
this is what I have in the bin folder !!

![bin folder](https://user-images.githubusercontent.com/55691088/83512560-0c555980-a4d9-11ea-9eab-50d99025fc58.PNG)

**Comment by DirtyRacer1337** [comment]
You need this files
![2020-06-02_17-59-23](https://user-images.githubusercontent.com/17379553/83512761-e2ad2a00-a4fa-11ea-982c-85415769c618.png)

**Comment by mukalaaa** [comment]
you sir are a wizard !! 
it works !! Thank you so much 

I know you muse be fed up with me, but are there any plans to support other sites at least the well knwn ones as a start such realitykings, bangbros...etc 

again, Thank you so much.... please please tell me is there a way to donate to help support this plugin? OMG, the jellyfin/emby community will flip out now. they have asking for an adult plugin for years 

also, may I ask one more question... as you know jellyfin is a fork of emby... yet when i tried your plugin with emby it did not even show up in the plugins list!!

any idea why?

**Comment by DirtyRacer1337** [comment]
I added some site, check sitelist. Still need to think about what site should be added first.
I think Jellyfin and Emby not compatible, maybe I'm wrong, I'm not tried it.
I added sponsor button, you can send donate if you want.
